libs/appwrite/src/lib/permissions/getUserPermissions.ts

```diff
@@ -0,0 +1,21 @@
+import { getAccount } from '../auth';
+import { getTeamMembership } from '../memberships';
+
+export const getUserPermissions = async ({
```

Comment:
whats stopping you from just defining arguments like 
```
const getUserPermissions = async (teamName: string) => {
... 
```

Reply:
More preference but I’ve changed it